### PR TITLE
Ayr 548/static page template design

### PR DESCRIPTION
**Pull request by zaiziclebby**
## Changes in this PR

- Created footer content pages and corresponding scss files. 
- Styled footer content pages according to. Terry's designs, which he has signed off on. 
- Accessibility page is the only page that has content so have only written tests for that page. 

## JIRA ticket
https://national-archives.atlassian.net/browse/AYR-548
### After
Screenshot 2024-01-04 at 09 42 16

- [ ] Requires env variable(s) to be updated
Screenshot 2024-01-04 at 09 41 56

### CODE REVIEW

File: app/templates/main/accessibility.html [hunk 1]
```diff
@@ -2,239 +2,177 @@
...
-                <li>navigate most of the website using speech recognition software</li>
-                <li>
-                    listen to most of the website using a screen reader (including the most recent versions of JAWS, NVDA and
-                    VoiceOver)
-                </li>
-            </ul>
-            <p class="govuk-body">We’ve also made the website text as simple as possible to understand.</p>
-            <p class="govuk-body">
-                <a class="govuk-link" href="https://mcmw.abilitynet.org.uk/">AbilityNet</a> has advice on making your device easier to use if you have a disability.
+    <main class="govuk-main-wrapper govuk-main-wrapper--terms-of-use">
```
Comment (anthonyhashemi) [inline app/templates/main/accessibility.html:6]: I know we spoke yesterday, but can you confirm where this content from the accessibility page came from? Is this the actual content we will put in, or just a placeholder?

Comment (zaiziclebby) [reply app/templates/main/accessibility.html:6]: here - https://national-archives.atlassian.net/browse/AYR-497 @anthonyhashemi

File: app/templates/base.html [hunk 2]
```diff
@@ -199,7 +211,7 @@
                     <h2 class="govuk-visually-hidden">Support links</h2>
                     <ul class="govuk-footer__inline-list">
                         <li class="govuk-footer__inline-list-item">
-                            <a class="govuk-footer__link" href="#">How to use this service</a>
+                            <a class="govuk-footer__link" href="{{ url_for('main.how_to_use') }}">How to use this service</a>
```
Comment (anthonyhashemi) [inline app/templates/base.html:214]: thanks for fixing this

**Review by anthonyhashemi — APPROVED** [review]
Looks good to me. I was thinking it could be nice to extract a base static content template if possible which can be shared across these if there is a consistent top level structure they share.

File: app/templates/main/privacy.html [hunk 3]
```diff
@@ -1,18 +1,71 @@
...
+                </p>
+                <p class="govuk-body">
+                    Odio aenean sed adipiscing diam donec adipiscing tristique risus nec. Donec massa sapien faucibus et molestie ac. Faucibus ornare suspendisse sed nisi lacus sed viverra tellus in. A lacus vestibulum sed arcu non odio. Pharetra convallis posuere morbi leo urna molestie at elementum eu. Malesuada nunc vel risus commodo viverra maecenas. Sollicitudin nibh sit amet commodo nulla facilisi nullam. In egestas erat imperdiet sed euismod nisi porta lorem. Nisi porta lorem mollis aliquam ut porttitor leo a diam. Etiam non quam lacus suspendisse faucibus. Morbi tristique senectus et netus. Bibendum arcu vitae elementum curabitur vitae. Scelerisque eu ultrices vitae auctor eu augue ut lectus arcu. Lacus luctus accumsan tortor posuere ac ut consequat.
+                </p>
+                <div class="terms-of-use-back-to-top">
```
Comment (anthonyhashemi) [inline app/templates/main/privacy.html]: this class name seems like a copy paste carry over

File: app/templates/main/how-to-use-this-service.html [hunk 4]
```diff
@@ -1,4 +1,71 @@
...
+                </p>
+                <p class="govuk-body">
+                    Odio aenean sed adipiscing diam donec adipiscing tristique risus nec. Donec massa sapien faucibus et molestie ac. Faucibus ornare suspendisse sed nisi lacus sed viverra tellus in. A lacus vestibulum sed arcu non odio. Pharetra convallis posuere morbi leo urna molestie at elementum eu. Malesuada nunc vel risus commodo viverra maecenas. Sollicitudin nibh sit amet commodo nulla facilisi nullam. In egestas erat imperdiet sed euismod nisi porta lorem. Nisi porta lorem mollis aliquam ut porttitor leo a diam. Etiam non quam lacus suspendisse faucibus. Morbi tristique senectus et netus. Bibendum arcu vitae elementum curabitur vitae. Scelerisque eu ultrices vitae auctor eu augue ut lectus arcu. Lacus luctus accumsan tortor posuere ac ut consequat.
+                </p>
+                <div class="terms-of-use-back-to-top">
```
Comment (anthonyhashemi) [inline app/templates/main/how-to-use-this-service.html]: and again here

File: app/templates/main/terms-of-use.html [hunk 5]
```diff
@@ -1,18 +1,71 @@
...
+                </p>
+                <p class="govuk-body">
+                    Odio aenean sed adipiscing diam donec adipiscing tristique risus nec. Donec massa sapien faucibus et molestie ac. Faucibus ornare suspendisse sed nisi lacus sed viverra tellus in. A lacus vestibulum sed arcu non odio. Pharetra convallis posuere morbi leo urna molestie at elementum eu. Malesuada nunc vel risus commodo viverra maecenas. Sollicitudin nibh sit amet commodo nulla facilisi nullam. In egestas erat imperdiet sed euismod nisi porta lorem. Nisi porta lorem mollis aliquam ut porttitor leo a diam. Etiam non quam lacus suspendisse faucibus. Morbi tristique senectus et netus. Bibendum arcu vitae elementum curabitur vitae. Scelerisque eu ultrices vitae auctor eu augue ut lectus arcu. Lacus luctus accumsan tortor posuere ac ut consequat.
+                </p>
+                <div class="terms-of-use-back-to-top">
```
Comment (anthonyhashemi) [inline app/templates/main/terms-of-use.html]: this class name doesnt seem to be defined. Should you be using the __back-to-top style you created?

**Pull request merged**